specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2022-09-04/redhatopenshift.json

```diff
@@ -0,0 +1,2441 @@
+          "type": "string"
+        }
+      }
+    },
+    "SyncIdentityProviderProperties": {
+      "description": "SyncSetProperties represents the properties of a SyncSet",
+      "type": "object",
+      "properties": {
+        "resources": {
+          "type": "string"
+        }
+      }
+    },
+    "SyncIdentityProviderUpdate": {
+      "description": "SyncIdentityProvider represents a SyncIdentityProvider",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SyncIdentityProviderProperties",
+          "description": "The SyncIdentityProvider Properties",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData",
+          "description": "The system meta data relating to this resource.",
+          "readOnly": true
+        }
+      }
+    },
+    "SyncSet": {
+      "description": "SyncSet represents a SyncSet for an Azure Red Hat OpenShift Cluster.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SyncSetProperties",
+          "description": "The Syncsets properties",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData",
+          "description": "The system meta data relating to this resource.",
+          "readOnly": true
+        }
+      }
+    },
+    "SyncSetList": {
+      "description": "SyncSetList represents a list of SyncSets",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of syncsets.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SyncSet"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "SyncSetProperties": {
+      "description": "SyncSetProperties represents the properties of a SyncSet",
+      "type": "object",
+      "properties": {
+        "resources": {
+          "description": "Resources represents the SyncSets configuration.",
+          "type": "string"
+        }
+      }
+    },
+    "SyncSetUpdate": {
+      "description": "SyncSet represents a SyncSet for an Azure Red Hat OpenShift Cluster.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SyncSetProperties",
+          "description": "The Syncsets properties",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
```

Comment:
>systemData [](http://example.com/codeflow?start=9&length=10)

SyncSetUpdate is defined as a parameter and it doesnt make sense for paramaters to have readOnly properties

Reply:
@rkmanda It is our intention that this systemData property is the Azure system metadata. In that context, should the systemData property be omitted from the swagger entirely?
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
I looked in the repo for other users of systemData, and our definitions seem to match theirs. Compare with adp.json [DataPoolPatch](https://github.com/Azure/azure-rest-api-specs/blob/4ac371de344db713847ff4d2ac1eae1e2223179a/specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2021-11-01-preview/adp.json#L828) and used in a [PATCH method here](https://github.com/Azure/azure-rest-api-specs/blob/4ac371de344db713847ff4d2ac1eae1e2223179a/specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2021-11-01-preview/adp.json#L515).